server/helpers/mailHelper/checklistReminders/createChecklistArray.js

```diff
@@ -0,0 +1,34 @@
+const generateListRow = require("./generateListRow");
+
+
+const createChecklistArray = ({
+  checklist, hostEmail, internEmail,
+}) => {
+  const checklistHtmlRows = checklist.forEach((checklistQuestion) => {
```

Comment:
this should be `map` not forEach 
```suggestion
  return checklist.map((checklistQuestion) => {
```